README.md

```diff
@@ -25,3 +25,81 @@ Synopsis:
 ---------
 
 See the docstrings in `textgrid.py`
+
+Example:
+---------
+This is a simple example of reading a TextGrid file. 
+```
+import textgrid
+
+# Read TextGrid object from file
+tg = textgrid.TextGrid.fromFile('test.TextGrid')
+
+# Interval example
+print ("------- Interval example -------")
+print (tg[0]) # item[1]
```

Comment:
While I would personally omit these comments altogether (I don't find them helpful) if you want to keep them them should match the syntax of the TextGrid file...e.g., `item [1]` not `item[1]`.